src/panels/config/helpers/forms/ha-schedule-form.ts

```diff
@@ -221,14 +221,16 @@ class HaScheduleForm extends LitElement {
         start.setDate(start.getDate() + distance);
         start.setHours(
           parseInt(item.from.slice(0, 2)),
-          parseInt(item.from.slice(-2))
+          parseInt(item.from.slice(3, 5)),
```

Comment:
this is not correct:

```
"9:30".slice(3,5) => '0'
"12:30".slice(3,5) => '30'
```

Reply:
As the value of my item is: `{from: '9:00', to: '13:30'}` so the format depends on the locale of the browser

Reply:
We can use:
```
item.from.split(":")[1];
```

Reply:
Changed to split() instead of slice().

Note: the root cause of the problem is the existence of the "seconds" (3rd) token in the time string. It doesn't exist when creating a new schedule, but it does exist when retrieving the config of a schedule.